app_sensor_module/src/sensor_module.h

```diff
@@ -3,18 +3,12 @@
 
 #include <launch_core/net/net_common.h>
 #include <launch_core/types.h>
-
 #include <stdbool.h>
 
-#define SENSOR_MODULE_IP_ADDR BACKPLANE_IP(SENSOR_MODULE_ID, 2, 1)
+#define SENSOR_MODULE_IP_ADDR             BACKPLANE_IP(SENSOR_MODULE_ID, 2, 1)
```

Comment:
?

Reply:
for some reason my ide really wants to align the defines lol, will disable that